docs/faq/README.md

```diff
@@ -52,14 +52,19 @@ System Cron:
 * Check if the **oc_jobs** table has a **reserved_at** entry with a value other than 0. If it does for whatever reason, set it to 0. You can check this by executing:
 
   ```sql
-  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater');
+  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\News\Cron\Updater","run"]' OR class = 'OCA\News\Cron\Updater');
   ```
+_For newer News versions please also use_
+SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\News\Cron\UpdaterJob","run"]' OR class = 'OCA\News\Cron\UpdaterJob');
```

Comment:
There should never be an `argument = '["OCA\News\Cron\UpdaterJob","run"]`  since the `BackgroundJob\Legacy` class isn't used in newer NC versions.

Reply:
I just copy/paste it over.

I'll paste the stuff I am usually using.